go/enclave/storage/cache_service.go

```diff
@@ -302,6 +302,7 @@ func (cs *CacheService) ReadSequencerIDs(_ context.Context, onCacheMiss func() (
 			return nil, err
 		}
 	}
+	cs.logger.Info("returning sequencer IDs from cache")
```

Comment:
did you intend to log the ids?

Reply:
No, they are logged directly after, this just points to where we get them from.